### PR TITLE
Update file naming strategy

### PR DESCRIPTION
**Pull request by nschcolnicov**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/custom/filterdifferentialtable/main.nf [hunk 1]
```diff
@@ -3,9 +3,7 @@ process CUSTOM_FILTERDIFFERENTIALTABLE {
     label 'process_single'
 
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pandas:1.5.2' :
-        'biocontainers/pandas:1.5.2' }"
+    container "community.wave.seqera.io/library/pandas_python:24935c20d1a97271"
```
Comment (SPPearce) [inline modules/nf-core/custom/filterdifferentialtable/main.nf:6]: Please add the singularity container too (there is a specific format that is available somewhere but I can't find immediately, remind me on Monday).

**Review by SPPearce — COMMENTED** [review]
Ummm, why are you changing the prefix in this way?

**Review by pinin4fjords — CHANGES_REQUESTED** [review]
I think this is based on a misunderstanding in discussion on differentialabundance. I asked to maintain some file names in the test outputs, but the way to do that is by using a closure on ext.prefix, not like this.

File: modules/nf-core/custom/filterdifferentialtable/main.nf (hunk 1 above)
Comment (nschcolnicov) [reply modules/nf-core/custom/filterdifferentialtable/main.nf:6]: Hi @SPPearce, please point to me towards the format that you are expecting here. Having the container with this format works for singularity too.

**Comment by nschcolnicov** [comment]
> I think this is based on a misunderstanding in discussion on differentialabundance. I asked to maintain some file names in the test outputs, but the way to do that is by using a closure on ext.prefix, not like this.

Hi @pinin4fjords why do you say that it should be done using a closure on ext.prefix? Isn't this the prefered strategy for naming files for this module in general? Or do we only want to label the files this way in differentialabundance? This strategy doesn't interfere with any other strategy that the user might prefer through ext.prefix

**Comment by nschcolnicov** [comment]
PR no longer needed

**Pull request closed without merging**